ardupilot/source/index.rst

```diff
@@ -1,13 +1,14 @@
 .. _home:
 
+=========
+ArduPilot
+=========
+
 ..  raw:: html
     <h1 style="text-align:center;">ArduPilot</h1>
 
-    <p style="text-align:center;color:red;"><strong>
```

Comment:
I do not want this removed...its important to stress up front these sections

Reply:
Will update shortly